README.md

```diff
@@ -4,11 +4,30 @@ A collection of classes, modules, macros and standard library extensions to simp
 
 Each tool is small, independent and generic.
 To use a tool, explicitly `require` where needed.
+
+```crystal
+require "inactive_support/<tool name>"
```

Comment:
`args` is very very handy but quite a surprise when first encountered.
It might make sense for a particular project to have the kernel extension (but it should be documented), and for others, used via an explicit include of a module containing the macro (or off the module directly).
These could be accessed via two different require paths.

this could be worth saving for another PR, but given this has quite a general title, I thought I may as well raise it here.